integration_tests/two_sided_pay_as_clear_market.feature

```diff
@@ -27,9 +27,7 @@ Feature: Two sided pay_as_clear market tests
      When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a for 2 hrs
      Then there are files with offers, bids & market_clearing_rate for every area
 
-  # TODO: re-enable in context of bug/D3ASIM-3534
```

Comment:
@Hassan754 I remember you disabling multiple integrationtests, or is my memory failing me and it was just this one?

Reply:
True, a global search with the ticket number will expose them

Reply:
![Screen Shot 2021-08-11 at 12 15 07](https://user-images.githubusercontent.com/33482427/129012314-0376af60-8cca-440a-a918-79109ecf0be5.png)

Reply:
Had also a look at all @disabled flags and I dont see any more.